drivers/sensor/bosch/bmm350/Kconfig

```
@@ -0,0 +1,60 @@
+# BMM350 Geomagnetic sensor configuration options
+
+# Copyright (c) 2024 Bosch Sensortec GmbH
+
+# SPDX-License-Identifier: Apache-2.0
+
+
+menuconfig BMM350
+	bool "BMM350 I2C Geomagnetic Chip"
+	default y
+	depends on DT_HAS_BOSCH_BMM350_ENABLED
+	select I2C
+	help
+	  Enable driver for BMM350 I2C-based Geomagnetic sensor.
+if BMM350
```

Nit `if` statements need to have empty newlines before and after

```suggestion
	  Enable driver for BMM350 I2C-based Geomagnetic sensor.

if BMM350
```